packages/api-server/package.json

```diff
@@ -54,9 +54,9 @@
     "access": "public"
   },
   "dependencies": {
-    "@dendronhq/common-all": "^0.108.0",
-    "@dendronhq/common-server": "^0.108.0",
-    "@dendronhq/engine-server": "^0.108.0",
+    "@dendronhq/common-all": "^0.108.1",
```

Comment:
Can we revert these patch updates? Else we'll get merge conflicts with release 109.